lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,744 @@
+    choices: [ absent, present ]
+    type: str
+  data_type:
+    description:
+    - Specifies the data type of the sequence. Valid types are bigint, integer, and smallint. bigint is the default.
+      The data type determines the default minimum and maximum values of the sequence.
+    - Supported from PostgreSQL 10.
+    required: false
+    choices: [ bigint, integer, smallint ]
+    type: str
+  increment:
+    description:
+    - Increment specifies which value is added to the current sequence value to create a new value.
+    - A positive value will make an ascending sequence, a negative one a descending sequence. The default value is 1.
+    required: false
+    type: int
+  minvalue:
+    description:
+    - Minvalue determines the minimum value a sequence can generate. The default for an ascending sequence is 1.
+      The default for a descending sequence is the minimum value of the data type.
+    required: false
+    type: int
+  maxvalue:
+    description:
+    - Maxvalue determines the maximum value for the sequence. The default for an ascending sequence is the maximum
+      value of the data type. The default for a descending sequence is -1.
+    required: false
+    type: int
+  start:
+    description:
+    - Start allows the sequence to begin anywhere. The default starting value is
+      I(minvalue) for ascending sequences and I(maxvalue) for descending ones.
+    required: false
+    type: int
+  cache:
+    description:
+    - Cache specifies how many sequence numbers are to be preallocated and
+      stored in memory for faster access. The minimum value is 1 (only one
+      value can be generated at a time, i.e., no cache), and this is also
+      the default.
+    required: false
+    type: int
+  cycle:
+    description:
+    - The cycle option allows the sequence to wrap around when the I(maxvalue) or I(minvalue) has been reached by an
+      ascending or descending sequence respectively. If the limit is reached, the next number generated will be the
+      minvalue or maxvalue, respectively.
+    - If false (NO CYCLE) is specified, any calls to nextval after the sequence has reached its maximum value will
+      return an error. False (NO CYCLE) is the default.
+    required: false
+    type: bool
+  cascade:
+    description:
+    - Automatically drop objects that depend on the sequence, and in turn all
+      objects that depend on those objects.
+    - Only used when I(state=absent).
+    required: false
+    type: bool
+  restrict:
+    description:
+    - Refuse to drop the sequence if any objects depend on it. This is the default.
+    - Only used when I(state=absent).
+    required: false
+    type: bool
+  rename_to:
+    description:
+    - The new name for the I(sequence).
+    - Works only for existing sequences.
+    required: false
+    type: str
+  owner:
+    description:
+    - Set the owner for the I(sequence).
+    - Works only for existing sequences.
+    required: false
+    type: str
+  schema:
```

Comment:
does it not conflict with:
```
  sequence:
    description:
    - The name (optionally schema-qualified) of the sequence.
```
For example if a user specifies a schema in both places?

Reply:
yes u are right, I will change it.

Reply:
change that in my last commit. Just removed the information of `(optionally schema-qualified)` usage. Schema can still be set during creation.